src/api/ApiResponder.php

```diff
@@ -144,7 +109,7 @@ protected function get_interface_response_createGame($interface, $args) {
         if (isset($retval)) {
             foreach ($playerIdArray as $playerId) {
                 if (isset($playerId)) {
-                    $interface->update_last_action_time($playerId, $retval['gameId']);
+                    $interface->player()->update_last_action_time($playerId, $retval['gameId']);
```

Comment:
I'm not convinced this actually works in this case --- it looks from the code below like in this case $interface is a BMInterfacePlayer object, so why do we need $interface->player() here?